removing autodeps suppression tags

azad-meta opened the pull request:
Summary: removing autodeps suppression tags

Differential Revision: D63335183

cccclai reviewed (approved):
Looks good to me but let's get review from @Gasoonjia too

azad-meta committed:
removing autodeps suppression tags (#5593)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/5593

removing autodeps suppression tags

Reviewed By: Gasoonjia, cccclai

Differential Revision: D63335183
